tests/location/UserLocationHooks.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { SetDependencyValue } from "@lib/dependencies"
+import {
+  LocationCoordinatesMocks,
+  UserLocationDependencyKeys,
+  UserLocationTrackingUpdate
+} from "@lib/location"
+import { useTrackUserLocation } from "@hooks/UserLocation"
+import { act, renderHook } from "@testing-library/react-native"
+
+describe("UserLocationHooks tests", () => {
+  describe("useUserLocation tests", () => {
+    beforeEach(() => jest.resetAllMocks())
+
+    const testLocation = {
+      coordinates: LocationCoordinatesMocks.NYC,
+      trackingDate: new Date()
+    }
+
+    const trackUserLocation = jest.fn()
+
+    const renderUseUserLocation = () => {
+      const wrapper = ({ children }: any) => (
+        <SetDependencyValue
+          forKey={UserLocationDependencyKeys.track}
+          value={trackUserLocation}
+        >
+          {children}
+        </SetDependencyValue>
+      )
+      return renderHook(useTrackUserLocation, { wrapper })
+    }
+
+    test("processes simple location updates", () => {
+      let sendLocationUpdate: (update: UserLocationTrackingUpdate) => void
```

Comment:
Are there any edge cases we'll need to test in this module, or is this just basic implementation